bin/download_anonymised

```diff
@@ -16,7 +16,7 @@ if [[ $DESTINATION_ENVIRONMENT == "local" ]]; then
 else
   kubectl exec deploy/laa-crime-application-store-app \
   -it -n laa-crime-application-store-$DESTINATION_ENVIRONMENT -- \
-  env ANONYMISED_DOWNLOAD_OUTPUT="${OUTPUT}" bundle exec rails anonymised:import\[ANONYMISED_DOWNLOAD_OUTPUT\]
+  env ANONYMISED_DOWNLOAD_OUTPUT="$OUTPUT" bundle exec rails anonymised:import\[ANONYMISED_DOWNLOAD_OUTPUT\]
```

Comment:
That's the opposite of what I meant :P Escaping it prevents something in the output causing it to be interpreted as part of the script, so you DO want them both to be `"${OUTPUT}"`